roboticslogger/file_handler.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+import errno
+import datetime
+
+def get_timestamped_filepath():
+    """ Gets a timestamped filepath for the logfiles """
+    return "{0}/Logs/RoboticsLogger/".format(os.path.expanduser('~')) 
```

Comment:
Interesting. I'm playing with it on Windows now, and it seems like python's os module actually provides this functionality under os.join. I'll try it again on my linux box then send a commit.